components/back-top/index.tsx

```diff
@@ -40,10 +40,6 @@ export interface BackTopProps {
 }
 
 export default class BackTop extends React.Component<BackTopProps, any> {
-  static defaultProps = {
-    visibilityHeight: 400,
-  };
```

Comment:
Why not use defaultProps?

Reply:
https://github.com/ant-design/ant-design-mobile/issues/1873